onebox.sh

```diff
@@ -0,0 +1,311 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT license. See LICENSE file on the project webpage for details.
+
+# Export all bash variable assignments (for use by sub-processes)
+# Write all commands to the console
+# Immmediately exit on error
+set -axe
+
+readonly DEFAULT_STRING="insecureDefault"
+readonly MSFT="microsoft"
+
+##########################
+# Script Defaults that can be overriden via
+# - parameter arguments OR
+# - assignment here
+##########################
+TEMPLATE_TYPE=fullstack # or devstack
+BRANCH_VERSIONS=edge    # or stable or edx
+DEFAULT_PASSWORD=
+
+MONGO_USER=oxamongoadmin
+MONGO_PASSWORD=$DEFAULT_STRING
+
+MYSQL_ADMIN_USER=root
+MYSQL_ADMIN_PASSWORD=
+
+MYSQL_USER=oxamysql
+MYSQL_PASSWORD=$DEFAULT_STRING
+
+EDXAPP_SU_USERNAME=edx_admin
+EDXAPP_SU_PASSWORD=$DEFAULT_STRING
+
+##########################
+# Settings
+##########################
+readonly BASE_URL=$HOSTNAME
+readonly LMS_URL=$BASE_URL # vanity
+readonly CMS_URL=$BASE_URL
+readonly PREVIEW_URL=$BASE_URL
+readonly PLATFORM_NAME="$MSFT Learning on $HOSTNAME"
+readonly EDXAPP_IMPORT_KITCHENSINK_COURSE=true
+readonly EDXAPP_ENABLE_THIRD_PARTY_AUTH=false
+readonly EDXAPP_SU_EMAIL="${EDXAPP_SU_USERNAME}@${MSFT}.com"
+readonly PLATFORM_EMAIL="$EDXAPP_SU_EMAIL"
+readonly EDX_BRANCH="open-release/ficus.master"
```

Comment:
the nearest tag in the upstream to our fork is open-release/ficus.1 so i'll switch it to that with my next commit. Specifically, our edx-platform fork lines up perfectly with open-release/ficus.1, but our fork from configuration is 13 commits before open-release/ficus.1